.github/actions/slack_release_results/action.yml

```diff
@@ -0,0 +1,90 @@
+name: 'send a slack release message'
+description: 'For a given channel ID and environment, send a slack message with the release status'
+inputs:
+  app_version:
+    description: "version of the app being deployed"
+    required: true
+  environment:
+    description: "deployment environment"
+    required: true
+  channel_id:
+    description: "Slack channel to send to"
+    required: true
+  deploy_outcome:
+    description: "The outcome of the deployment"
+    required: true
+  deployment_changelog:
+    description: "A log of previous deployments"
+    required: false
+    default: ""
+  slack_bot_token:
+    description: "Slack bot token"
+    required: true
+  k8s_deployment_name:
+    description: "Kubernetes deployment name"
+    required: true
+runs:
+  using: "composite"
+  steps:
+  - name: Set up Python
+    uses: actions/setup-python@v2
+    with:
+      python-version: '3.x'
+  - name: Sanitised first line of commit message
+    run: |
+      import os
+      import re
+      remove_chars='{}()><&*‘|=?;[]$–#~!.%"\\/:+,`-'
+      pattern = f'[{re.escape(remove_chars)}]'
+      commit_message=os.getenv('COMMIT_MESSAGE','')
+      # First line of commit message (and first 256 lines if longer)
+      if commit_message:
+        first_line=commit_message.split('\n')[0][:256]
+        sanitised_message = re.sub(pattern, '', first_line)
+        with open(os.getenv('GITHUB_ENV'), 'a') as gh_env:
+          gh_env.write(f'commit_message_first_line={sanitised_message}\n')
+        gh_env.close()
+    shell: python {0}
+    env:
+      COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+
+  - name: Slack - Send a message
+    id: slack-release-message
+    uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
+    with:
+      channel-id: ${{ inputs.channel_id }}
+      payload: |
+          {
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "*${{ github.inputs.k8s_deployment_name }}* version `${{ inputs.app_version }}` deploy to *${{ inputs.environment }}*"
```

Comment:
This is the change from github actions  (now uses k8s_deployment_name, rather than  github.event.repository.name )